recipe/build.sh

```diff
@@ -6,14 +6,19 @@ CMAKE_ARGS="$CMAKE_ARGS -DCURSES_INCLUDE_PATH=${PREFIX}/include -DBUILD_CursesDi
 CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_PREFIX_PATH=${PREFIX}"
 
 if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
+   if [[ "$target_platform" == osx-* ]] && [[ "$MACOSX_DEPLOYMENT_TARGET" == 11.* || "$MACOSX_DEPLOYMENT_TARGET" == "10.15" ]]; then
```

Comment:
@isuruf - do you want to add yourself as a maintainer here too?

Reply:
No, I don't. cc @conda-forge/core

Reply:
Isn't it more correct to ask @conda-forge/cmake?